selfservice/strategy/oidc/provider_generic_oidc.go

```diff
@@ -97,16 +109,31 @@ func (g *ProviderGenericOIDC) verifyAndDecodeClaimsWithProvider(ctx context.Cont
 	return &claims, nil
 }
 
+func verifyAudience(received []string, expected []string) error {
```

Comment:
While the code is simple enough, it would be good to still have a test so that noone can introduce a regression here ever.

Reply:
Have added test to `selfservice/strategy/oidc/strategy_test.go`